graph/service.go

```diff
@@ -12,8 +12,7 @@ import (
 func (s *TagStore) Install(eng *engine.Engine) error {
 	for name, handler := range map[string]engine.Handler{
 		"image_set":      s.CmdSet,
-		"image_tag":      s.CmdTag,
-		"tag":            s.CmdTagLegacy, // FIXME merge with "image_tag"
+		"tag":            s.CmdTagLegacy,
```

Comment:
yeah I was just thinking about it :) ... but had already pushed the code.
I ll fix it.